var/spack/repos/builtin/packages/hpctoolkit-externals/package.py

```diff
@@ -0,0 +1,45 @@
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class HpctoolkitExternals(Package):
+
+    """
+    HPCToolkit performance analysis tool has many prerequisites and
+    HpctoolkitExternals package provides all these prerequisites.
+    """
```

Comment:
The documentation tests are crashing because your docstring can't start with a newline character. Please format them like:
```python
class HpctoolkitExternals(Package):
    """HPCToolkit performance analysis tool has many prerequisites and
    HpctoolkitExternals package provides all these prerequisites."""
```